Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```
@@ -839,6 +837,12 @@ script:
       required: true
       isArray: true
       defaultValue: ""
+    - default: true
+      description: ''
+      isArray: true
+      name: cve
+      required: true
+      secret: false
```

why you added this argument?
```suggestion
    - default: true
      description: ''
      isArray: true
      name: cve
      required: true
      secret: false
```